Jenkinsfile.rh

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2017-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/nexus/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared']) _
+import com.sonatype.jenkins.pipeline.GitHub
+import com.sonatype.jenkins.pipeline.OsTools
+
+node('ubuntu-zion') {
+  def commitId, commitDate, version, imageId, branch, dockerFileLocations
+  def organization = 'sonatype',
+      gitHubRepository = 'docker-nexus-iq-server',
+      credentialsId = 'integrations-github-api',
+      imageName = 'sonatype/nexus-iq-server',
+      dockerHubRepository = 'nexus-iq-server'
+  GitHub gitHub
+
+  try {
+    stage('Preparation') {
+      deleteDir()
+      OsTools.runSafe(this, "docker system prune -a -f")
```

Comment:
I feel like this build may not need to bother doing any local `docker` stuff, so they can likely be removed from this jenkins file.

Reply:
@jflinchbaugh It's make sense, I'm going to refactor/ remove everything is not direct related with TriggerRedHatBuild.groovy call